app/src/androidTest/java/org/mozilla/reference/browser/helpers/TestHelper.kt

```diff
@@ -17,4 +18,11 @@ object TestHelper {
         appView.scrollTextIntoView(text)
         return appView
     }
+
+    fun getPermissionAllowID(): String {
+        return when (Build.VERSION.SDK_INT > Build.VERSION_CODES.P) {
+            true -> "com.android.permissioncontroller"
+            false -> "com.android.packageinstaller"
+        }
```

Comment:
Is this from Fenix? What is this doing?

Reply:
Yes, it returns the resource ID of an item based on the Android version